### ISSUE TITLE
Banner Bug- Keeps Buffs without area of banner 

### ISSUE THREAD

**Issue by Toshutkidup** [WarEmu/WarBugs#18916]
This is a continuation of the first banner bug I showed on stream that allows you to keep buffs while out of range of a banner. This one if a player places a banner and walks away from it some buffs will stay active for that player even tho player is out of range for banner. 

**Expected behavior and actual behavior:**
Banner Buffs should all be removed when out of range of banner

**Steps to reproduce the problem:**
I have dropped several banners and tried to walk away and many times ( NOT 100%) the buff(s) will stay 

I have a unlisted YT vid displaying issue bellow
https://youtu.be/_GUMRjSuy9Y

**Comment by lilimx** [comment]
Hey,

is this still a thing? I've tried to reproduce it, but no luck. Is it only the strenght tactic or did other tactics work weird?

**Comment by GamesBondd** [comment]
@Toshutkidup reopened

**Comment by Toshutkidup** [comment]
@lilimx 

yes this is still a issue and I have been able to duplicate it several times on streams. Its not ALWAYS the strength as it depends on what buffs the banner has, I have 2 banners for diff buffs. so it dependent on the banner / guild buffs.,

**Comment by Toshutkidup** [comment]
@lilimx

I have now also included another video evidence and this one I was able to have 2 Buff's apply and stay on em while I was walking and in combat. And I also used Rag doll , Stat Doll and .Get Stats to show my stats are being increased with no banner in range. 

This is a longer clip to show its actively enabled and working in ORVR combat, o please watch entire vid. 

Video link is below. 

https://www.twitch.tv/videos/1045699153

**Comment by Toshutkidup** [comment]
Well noone has inquired about this bug report but I have video of a Destro grp roaming ORVR using it.. so now its being exploited. Sorry no audio forgot to turn  on mic. 

YT Link:
https://youtu.be/AeYiZMbkZSk

**Comment by Toshutkidup** [comment]
I have recently re-tested on stream Friday April 8th 2022 and exploit / bug is still in game
